packages/login/src/modules/passwordUpdate/password/sagas.js

```diff
@@ -13,46 +14,41 @@ export const usernameSelector = state => state.passwordUpdate.dialog.username
 export const passwordSelector = state => state.passwordUpdate.password
 export const standaloneSelector = state => state.passwordUpdate.dialog.standalone
 
-function doRequest(data, username, action) {
+export function* storePassword(username, data) {
   const options = {
     method: 'POST',
-    body: JSON.stringify(data),
-    headers: new Headers({
-      'Content-Type': 'application/json'
-    }),
-    credentials: 'include'
+    acceptedStatusCodes: [400],
+    body: data
   }
-  return fetch(`${__BACKEND_URL__}/nice2/rest/principals/${username}/${action}`, options)
-}
 
-export function storePassword(username, data) {
-  return doRequest(data, username, 'password-update')
-    .then(resp => {
+  try {
+    const resp = yield call(requestSaga, `principals/${username}/password-update`, options)
+    if (resp) {
```

Comment:
In which case could `resp` be missing and what should we do in that case (currently, nothing happens)?